src/Aspire.Dashboard/Model/DashboardCommandExecutor.cs

```diff
@@ -36,9 +38,25 @@ public async Task ExecuteAsync(ResourceViewModel resource, CommandViewModel comm
             _executingCommands.Add(executingCommandKey);
         }
 
+        await telemetryService.InitializeAsync().ConfigureAwait(false);
```

Comment:
Hmmm, I don't like how telemetry service needs to be initialized everywhere just in case it hasn't already been initialized.

This is a scoped service so it can't just be initialized once when the app starts. We need to find an event after a circuit starts and initialize there. Don't try and fix this now, but I want to make this better.